second-analysis-steps/code/building-decays/01.historical.py

```diff
@@ -65,23 +65,20 @@
 # Create an ntuple
 dtt = DecayTreeTuple('TupleDstToD0pi_D0ToKpi')
```

Comment:
This is a minor thing, but for consistency the tuple name should be `TupleDstToD0pi_D0ToKK`